ironfish/src/primitives/note.ts

```diff
@@ -29,7 +29,7 @@ export class Note {
     // skip owner public address
     reader.seek(PUBLIC_ADDRESS_LENGTH)
 
-    this._assetIdentifier = reader.readBytes(PUBLIC_ADDRESS_LENGTH, true)
```

Comment:
the asset identifier length is define with another constant. Its defined here https://github.com/iron-fish/ironfish/blob/fe19ede6ca0631f42441ec4a52698966f31f163c/ironfish-zkp/src/constants.rs#L13
You will need to export in rust-nodejs package using napi, then you can refer it in ts.